app/models/batch_connect/app.rb

```diff
@@ -175,6 +175,14 @@ def session_view
       file.read if file.file?
     end
 
+    # Paths to custom javascript files
+    # @return [Pathname] paths to custom javascript files that exist
+    def custom_javascript_files
+      files = [root.join("form.js")]
+      files << sub_app_root.join("#{sub_app}.js")
```

Comment:
Why permit loading a second script? Do you just want to give developers the option of something other than `form.js`? Or do you envision a use case where the second script is necessary?